### ISSUE TITLE
Difficulty getting model to perform decently on the validation set

### ISSUE THREAD

**Issue by maltium** [facebookresearch/wav2letter#447]
I'm trying to train a model based on the WSJ receipe but I'm getting the behaviour shown below. For this particular run I used Librispeech and trained on 10 hours of data. I've made attempts with bigger datasets (40hr and 80hr) and although the numbers are slightly different, I'm getting this consistent behaviour:

Initially validation loss is lower that training loss, eventually training loss catches up to the validation loss and the validation loss plateaus (and gradually starts increasing) even though the WER continues to decrease slightly.

Also did some runs with different learning rates and dropouts but did not see any significant changes. *train.cfg* and *network.arch* are identical to those in the WSJ receipe.

As an aside I'm noticing that the model.bin file on some systems is around 63mb while on others it's around 142mb. The network architecture looks identical when printed out to the console.

Please could anyone offer some insight on how to get improved results? These results are clearly way off what should be expected, so something is fundamentally wrong.

![loss](https://user-images.githubusercontent.com/4674453/69678965-474c7f80-10a7-11ea-8aae-59698a8d532e.png)

![ter](https://user-images.githubusercontent.com/4674453/69678982-50d5e780-10a7-11ea-9fd8-2f2bc92ddb93.png)

![wer](https://user-images.githubusercontent.com/4674453/69678991-54696e80-10a7-11ea-9878-96d67ecc80d6.png)

**Comment by lunixbochs** [comment]
This doesn't seem that strange to me, using 10 hours of audio on wsjarch won't give you anything close to the librispeech numbers in the whitepaper. I put 3000 hours of audio into wsjarch and only got librispeech down to 5.5%

**Comment by jacobkahn** [comment]
@maltium — as @lunixbochs says, using this little labeled audio is very difficult. It's certainly an open research problem in the field.

We're slowly releasing papers/code for some of our low-resource work in the semi-supervised setting, so keep an eye out for that over the next few weeks.
